src/components/SearchableDropdown/__snapshots__/SearchableDropdown.stories.storyshot

```diff
@@ -19,17 +19,17 @@ exports[`storybook Storyshots Components/Searchable Dropdown Custom Option 1`] =
     >
       <button
         aria-disabled={false}
-        className="leafygreen-ui-26sqyb css-mbqq81-StyledButton e1yw4j302"
+        className="lg-ui-button-0000 leafygreen-ui-n2d19n css-mbqq81-StyledButton e1yw4j302"
```

Comment:
Noticing this weird padding on this component. 
<img width="1487" alt="image" src="https://user-images.githubusercontent.com/4605522/202014974-0bfc1686-9aa1-4fa3-b1cb-1c6976183bed.png">

Reply:
<img width="1458" alt="image" src="https://user-images.githubusercontent.com/4605522/202015635-439ed67d-d9e3-4f11-9da9-164aed01470b.png">
Even for smaller strings